include/model.php

```diff
@@ -747,28 +769,56 @@ public function get_terms_with_no_lang( $taxonomies, $limit ) {
 	 *
 	 * @since 1.2
 	 * @since 3.4 Moved from PLL_Admin_Model class.
-	 * @since 3.4 Change method signature.
-	 *
-	 * @param int $limit Max number of posts or terms to return. Defaults to -1 (no limit).
-	 * @return void
+	 *            Removed `$limit` parameter, added `$lang` and `$types` parameters.
+	 *            Returns the types that may still have non-translated objects.
 	 *
-	 * @phpstan-param -1|positive-int $limit
+	 * @param PLL_Language|null $lang  Optional. The language to assign to objects. Defaults to `null` (default language).
+	 * @param string[]          $types Optional. Types to handle (@see PLL_Translatable_Object::get_type()). Defaults
+	 *                                 to an empty array (all types).
+	 * @return string[] The types that may still have non-translated objects.
```

Comment:
Not sure, but returning `void` is enough, right?

Reply:
No, that's what makes the method recursive while doing less db queries.